### PR TITLE
Updated the Project Folder Structure and added an assests folder that contains images

### PR DESCRIPTION
**Pull request by silasade**

### CODE REVIEW

**Review by SamuelOmodele — APPROVED** [review]
Folder structure has been modified. 

Approved.

File: src/Assests/Images/Ellipse 35.svg
Comment (a-k-o-r-e-d-e) [inline src/Assests/Images/Ellipse 35.svg:1]: @silasade typo in assests should be 'assets'

File: src/Assests/Images/Group 1171275826.png
Comment (a-k-o-r-e-d-e) [inline src/Assests/Images/Group 1171275826.png:1]: Give this file a proper descriptive name

File: src/Assests/Images/teenyicons_tick-circle-solid.png
Comment (a-k-o-r-e-d-e) [inline src/Assests/Images/teenyicons_tick-circle-solid.png:1]: Is this an icon? why is ut being imported as an image?

File: src/Global Components/success-page/Success.tsx
Comment (a-k-o-r-e-d-e) [inline src/Global Components/success-page/Success.tsx:1]: Is this page or a modal, the name should reflect that

File: src/Pages/NormalUsers/Sidebar/Sidebar.css
Comment (a-k-o-r-e-d-e) [inline src/Pages/NormalUsers/Sidebar/Sidebar.css:1]: The Sidebar is a component, not a page @silasade

File: src/Pages/Selectquiz/quiz-card/QuizCard.tsx
Comment (a-k-o-r-e-d-e) [inline src/Pages/Selectquiz/quiz-card/QuizCard.tsx:1]: QUiz card is a component

File: README.md
Comment (a-k-o-r-e-d-e) [inline README.md:1]: Fix the conflicts with the ReadMe.md file

File: src/Assests/Images/teenyicons_tick-circle-solid.png
Comment (SamuelOmodele) [reply src/Assests/Images/teenyicons_tick-circle-solid.png:1]: That was the "green tick" image used in the figma design, so I used it.

File: src/Assests/Images/Ellipse 35.svg
Comment (silasade) [reply src/Assests/Images/Ellipse 35.svg:1]: Done

File: src/Assests/Images/teenyicons_tick-circle-solid.png
Comment (a-k-o-r-e-d-e) [reply src/Assests/Images/teenyicons_tick-circle-solid.png:1]: Use a proper icon from the [react-icons](https://react-icons.github.io/react-icons/) library, if you can't find it, look for something that looks as close as possible 

cc: @SamuelOmodele

File: README.md
Comment (silasade) [reply README.md:1]: Done.

File: src/Pages/NormalUsers/Sidebar/Sidebar.css
Comment (silasade) [reply src/Pages/NormalUsers/Sidebar/Sidebar.css:1]: Corrected

File: src/Global Components/success-page/Success.tsx
Comment (silasade) [reply src/Global Components/success-page/Success.tsx:1]: Created a subfolder with the name modals which contains all the modals

File: src/Assests/Images/Group 1171275826.png
Comment (a-k-o-r-e-d-e) [reply src/Assests/Images/Group 1171275826.png:1]: @silasade @SamuelOmodele who is to fix this?

**Comment by a-k-o-r-e-d-e** [comment]
Merging this , so as not to delay the rest of you while we wait for @SamuelOmodele changes in another PR

**Pull request merged**